src/NServiceBus.Metrics/Probes.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace NServiceBus.Metrics
+{
+    /// <summary>
+    /// This class provides identifiers for all the probes provided by this package.
+    /// </summary>
+    public static class Probes
```

Comment:
Ah so with that like @dvdstelt we might even be able to just remove all attributes right since we can enumerate all the constants here with cecil?